test/functional/app/api/test_transfer_usage_api.py

```diff
@@ -30,3 +30,32 @@ def test_transfer_usage_api(test_client, authed_sempo_admin_user,name,icon,trans
                                 content_type='application/json', follow_redirects=True)
 
     assert response.status_code == status_code
+
+    # Ensure that the posting a duplicate transfer ussage results in a 400
+    response = test_client.post('/api/transfer_usage/',
+                                headers=dict(Authorization=auth_token + '|' + tfa_token, Accept='application/json'),
+                                data=json.dumps(dict(name=name, icon=icon, translations=translations)),
+                                content_type='application/json', follow_redirects=True)
+
+    assert response.status_code == 400
+
+
+def test_transfer_usage_api_get(test_client, authed_sempo_admin_user):
+    """
+    GIVEN a Flask application
+    WHEN the '/api/transfer_usage/' page is requested (GET)
+    THEN check the response has status 200 and a list
+    """
+    authed_sempo_admin_user.is_activated = True
+    authed_sempo_admin_user.TFA_enabled = True
+    authed_sempo_admin_user.set_held_role('ADMIN', 'admin')
+    auth_token = authed_sempo_admin_user.encode_auth_token().decode()
+    tfa_token = authed_sempo_admin_user.encode_TFA_token(9999).decode()
+
+    response = test_client.get('/api/transfer_usage/',
+                               headers=dict(
+                                   Authorization=auth_token + '|' + tfa_token, Accept='application/json'),
```

Comment:
we actually have a helper function makes this a lot easier 
`from server.utils.auth import get_complete_auth_token`

Then all of the lines above, including activating etc _should_ be possible to delete.